tools/settings.js

```diff
@@ -5,10 +5,11 @@
  * file and the package environment.
  */
 
-'use strict';
+import * as fs from 'node:fs';
+import * as path from 'node:path';
```

Comment:
While writing this I realized we didn't have a convention about whether to use the `node:` prefix or not and whether it was considered preferred practice. https://github.com/nodejs/node/issues/38343 suggests that using the prefix for built-in modules is preferred.